<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPixivIndexLimitsDialog</class>
 <widget class="QDialog" name="CPixivIndexLimitsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Keywords</string>
        </property>
        <property name="buddy">
         <cstring>editKeywords</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="editKeywords">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Search mode</string>
        </property>
        <property name="buddy">
         <cstring>comboMode</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboMode"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Rating</string>
        </property>
        <property name="buddy">
         <cstring>comboRating</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboRating"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Max results count</string>
        </property>
        <property name="buddy">
         <cstring>spinMaxCount</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spinMaxCount">
        <property name="specialValueText">
         <string>All</string>
        </property>
        <property name="maximum">
         <number>9999999</number>
        </property>
        <property name="singleStep">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="checkDateFrom">
        <property name="text">
         <string>Date from</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDateEdit" name="dateFrom">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="checkDateTo">
        <property name="text">
         <string>Date to</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDateEdit" name="dateTo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Text length</string>
        </property>
        <property name="buddy">
         <cstring>comboLength</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="comboLength"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Language</string>
        </property>
        <property name="buddy">
         <cstring>comboLanguage</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="comboLanguage"/>
      </item>
      <item row="8" column="1">
       <widget class="QCheckBox" name="checkOriginalOnly">
        <property name="text">
         <string>Only original works</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editKeywords</tabstop>
  <tabstop>comboMode</tabstop>
  <tabstop>comboRating</tabstop>
  <tabstop>spinMaxCount</tabstop>
  <tabstop>checkDateFrom</tabstop>
  <tabstop>dateFrom</tabstop>
  <tabstop>checkDateTo</tabstop>
  <tabstop>dateTo</tabstop>
  <tabstop>comboLength</tabstop>
  <tabstop>comboLanguage</tabstop>
  <tabstop>checkOriginalOnly</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>checkDateFrom</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateFrom</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkDateTo</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateTo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CPixivIndexLimitsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CPixivIndexLimitsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
